src/rt-app_parse_config.c

```diff
@@ -836,7 +836,8 @@ static sched_data_t *parse_sched_data(struct json_object *obj, int def_policy)
 	tmp_data.deadline *= 1000;
 
 	/* Check if we found at least one meaningful scheduler parameter */
-	if (tmp_data.prio != -1 || tmp_data.runtime || tmp_data.period || tmp_data.period) {
```

Comment:
Haha. We were checking for period twice here :D

So you're actually fixing a typo here. Worth mentioning in the commit message but that's really a minor nit only worth doing if this series gets updated. But not worth updating it only for that.

Reply:
This chunk is already in a dedicated patch, do you mean that the changelog of that patch is not clear enough?

Reply:
I guess it is clear. Feel free to ignore this.